src/gobject.cc

```diff
@@ -400,7 +400,7 @@ Local<Function> MakeClass(GIBaseInfo *info) {
     return tpl->GetFunction ();
 }
 
-Local<Value> WrapperFromGObject(GObject *gobject) {
+Local<Value> WrapperFromGObject(GObject *gobject, GIBaseInfo *object_info) {
```

Comment:
Why is this changed if it's not used anywhere?

Reply:
It is now :) That was the missing code.